spyder/plugins/console/widgets/shell.py

```diff
@@ -640,7 +640,9 @@ class PythonShellWidget(TracebackLinksMixin, ShellBaseWidget,
     INITHISTORY = ['# -*- coding: utf-8 -*-',
                    '# *** Spyder Python Console History Log ***',]
     SEPARATOR = '%s##---(%s)---' % (os.linesep*2, time.ctime())
-    go_to_error = Signal(str)
+
+    # Signals
+    sig_go_to_error_requested = Signal(str)

```

Comment:
Please add a comment above this signal to describe what it does.

Reply:
Fixed